### ISSUE TITLE
I would like to submit the form in each step, any idea how to implement?

### ISSUE THREAD

**Issue by Zegoverno** [saini-g/react-step-progress#23]
I am also using react-hook-form, anyone has ideas to work with both? Thanks

**Comment by alexiz10** [comment]
I don't believe that is customizable out of the box. But you could copy the source code in `index.tsx` and customize it to your liking. On line 162, there is an onClick handler for the 'Next' Button, so you could start working from there.

EDIT: Just looked a little more into how the component works and I think you could do this in your validators. Maybe validate, if valid, submit data and return true. If not valid return false.
